src/main/osd/osd_elements.c

```diff
@@ -1014,8 +1015,18 @@ static void osdElementGpsHomeDirection(osdElementParms_t *element)
 {
     if (STATE(GPS_FIX) && STATE(GPS_FIX_HOME)) {
         if (GPS_distanceToHome > 0) {
```

Comment:
(technically, gpsLapTimerData.timerRunning will be ignored with GPS_distanceToHome == 0)

Also, does code below handle correctly case when distance to gate is zero?